content/en/tracing/trace_collection/otel_instrumentation/go.md

```diff
@@ -23,10 +26,10 @@ The following OpenTelemetry features are implemented in the Datadog library as n
 
 | Feature                               | Support notes                       |
 |---------------------------------------|------------------------------------|
-| [OpenTelemetry Context propagation][1]         | [Datadog distributed header format][9] is used instead. |
+| [OpenTelemetry Context propagation][1]         | [W3C and Datadog header formats][9] are used. |
```

Comment:
Oh another comment, maybe we can say that these are enabled by default. So taking the previous comment and this comment together, it would look like the following. What do you think?
```suggestion
| [OpenTelemetry Context propagation][1]         | [W3C Trace Context and Datadog header formats][9] are enabled, by default. |
```